designs/administrator/olm/convey-installing-operator/index.md

```diff
@@ -0,0 +1,45 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Convey Operator is Installing
+
+Currently when an operator is installed the user is taken to the **Installed Operators** list view, where it may take some time before the entry for the newly installed operator actually appears, particularly if the operator is installed globally and the user is viewing a namespace the operator will be copied to, and the CSV doesn't yet exist. This design gives the admin user immediate feedback that the operator is installing properly, and can surface errors that may occur during the installation process.
+
+## Installing an Operator
+
+![Install operator](img/1-1-install.png)
+- The user selects the operator they'd like to install and selected installation options (no changes to this screen.)
+
+![Installing operator](img/1-2x-installing-leave.png)
+- The user is now taken to the new installing screen where a spinner conveys the operator is currently being installed.
+- The status conditions reason and phase are shown and update in real time on the installing screen.
+- A link is included that allows the user to exit to the Installed Operators list.
+- The user could navigate away from this screen using the left nav, and return to it via the **Installed Operators** list view.
+
+![Installed operator list](img/y-1-list.png)
+- On the Installed Operators list, a row item would always appear (even if the CSV isn’t created in that project yet) that conveys the operator is installing.
+- A **View Install** link will be present when the operator is installing, that can return the user to the installing screen.
+- **Definition of 'operator installing':** Before CSV's status.phase reaches "Succeeded"
+- Values in columns may be unfilled as neccessary.
+
+![Installing operator](img/1-2-installing.png)
```

Comment:
It might be inconsistent with itself, but does it make sense to consider a popover here instead so the user isn't taken out of the context of the list view?